### ISSUE TITLE
[Feature request] Use a profile for activation of environment variables or enable passing it via cli

### ISSUE THREAD

**Issue by cheburakshu** [GoogleContainerTools/skaffold#8434]
One can run a skaffold profile using the following invocation:

`skaffold run -p profile_name`

When doing so, there are a lot of ops that can be done on the manifests, images and so on.

It would be quite nice to be able to define a set of environment variables that needs to activated, in addition to whatever is present in the current shell/env, when choosing to run using a profile.

Currently, the users are required to define all env variables outside skaffold, and skaffold will only use those, either for activation or for the lifecycle of the deployment. Skaffold cli does not even accept an input to .env file or similar.

I feel this bridge needs to be created to enable users to enjoy the fullest potential of skaffold profiles.

For comparison, see how `kubectl run` accepts it

https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#run
```
kubectl run NAME --image=image [--env="key=value"] [--port=port] [--replicas=replicas] [--dry-run=bool] [--overrides=inline-json] [--command] -- [COMMAND] [args...]
```

**Comment by aaron-prindle** [comment]
Tagging - @gsquared94 

`.env` file support was recently added to Skaffold and is available @ HEAD and in our next release `v2.2.0`:
https://github.com/GoogleContainerTools/skaffold/pull/8395

I think with that change this feature request is possibly already in Skaffold @ HEAD

**Comment by cheburakshu** [comment]
Fantastic! Thanks, I will close this.
